src/expressions/loop-expr.md

```diff
@@ -226,6 +241,15 @@ Example:
 
 A `break` expression is only permitted in the body of a loop, and has one of the forms `break`, `break 'label` or ([see below](#break-and-loop-values)) `break EXPR` or `break 'label EXPR`.
 
+## Label expressions
```

Comment:
Similarly, I might make this a little clearer by calling them "Labeled block expressions".